config/kcp/api_export.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: apis.kcp.dev/v1alpha1
```

Comment:
Can we not rename this file? Every other stream calls theirs `apiexport_<stream name>.yaml`. I don't see any benefit from renaming. Also, if KCP wants to just grab all of our apiexports, ours will at best be unclear based on the filename and at worst, conflict with others if they decide to copy our style

Reply:
Having the release prefix here when this is a resource that will be just deployed in the cluster under a different name doesn't make sense to me at all, but yes, can be named back to `apiexport_release.yaml`.

Reply:
They would be assuming too much and writing crappy code. In any case, renaming it back.